PHASE_1/API_SourceCode/src/App.ts

```diff
@@ -35,13 +36,15 @@ export default class App {
     const articleRouter = new ArticleRouter(this.articleService);
     const searchRouter = new SearchRouter(this.searchService);
     const reportRouter = new ReportRouter(this.reportService);
+    const reportSpecificRouter = new ReportSpecificRouter(this.reportService);
 
     this.ex.addRouters(
       // ... add routers here
       nameRouter,
       articleRouter,
       searchRouter,
-      reportRouter
+      reportRouter,
+      reportSpecificRouter
```

Comment:
for specific reports and articles, we shouldn't use a different router since they have the same parent path.